spec/test_app/app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
+  def root
```

Comment:
this is like *so* not that important but wanna do `index` here just so that was we expand this test suite we are using super vanilla railsy stuff?